contracts/impls/trustable/WitnetRequestBoardTrustableBase.sol

```diff
@@ -0,0 +1,694 @@
+                && _owner == _from
+        );
+    }
+
+
+    // ================================================================================================================
+    // --- Full implementation of 'IWitnetRequestBoardAdmin' ----------------------------------------------------------
+
+    /// Gets admin/owner address.
+    function owner()
+        public view
+        override
+        returns (address)
+    {
+        return _state().owner;
+    }
+
+    /// Transfers ownership.
+    function transferOwnership(address _newOwner)
+        external
+        virtual override
+        onlyOwner
+    {
+        address _owner = _state().owner;
+        if (_newOwner != _owner) {
+            _state().owner = _newOwner;
+            emit OwnershipTransferred(_owner, _newOwner);
+        }
+    }
+
+
+    // ================================================================================================================
+    // --- Full implementation of 'IWitnetRequestBoardAdminACLs' ------------------------------------------------------
+
+    /// Tells whether given address is included in the active reporters control list.
+    /// @param _reporter The address to be checked.
+    function isReporter(address _reporter) public view override returns (bool) {
+        return _acls().isReporter_[_reporter];
+    }
+
+    /// Adds given addresses to the active reporters control list.
+    /// @dev Can only be called from the owner address.
+    /// @dev Emits the `ReportersSet` event. 
+    /// @param _reporters List of addresses to be added to the active reporters control list.
+    function setReporters(address[] memory _reporters)
+        public
+        override
+        onlyOwner
+    {
+        for (uint ix = 0; ix < _reporters.length; ix ++) {
+            address _reporter = _reporters[ix];
+            _acls().isReporter_[_reporter] = true;
+        }
+        emit ReportersSet(_reporters);
+    }
+
+    /// Removes given addresses from the active reporters control list.
+    /// @dev Can only be called from the owner address.
+    /// @dev Emits the `ReportersUnset` event. 
+    /// @param _exReporters List of addresses to be added to the active reporters control list.
+    function unsetReporters(address[] memory _exReporters)
+        public
+        override
+        onlyOwner
+    {
+        for (uint ix = 0; ix < _exReporters.length; ix ++) {
+            address _reporter = _exReporters[ix];
+            _acls().isReporter_[_reporter] = false;
+        }
+        emit ReportersUnset(_exReporters);
+    }
+
+
+    // ================================================================================================================
+    // --- Full implementation of 'IWitnetRequestBoardReporter' -------------------------------------------------------
+
+    /// Reports the Witnet-provided result to a previously posted request. 
+    /// @dev Will assume `block.number` as the epoch number for the provided result.
```

Comment:
Is this comment true? `epoch` is set to 0, and I don't see any mentions of `block.number`.

Reply:
Sorry, not anymore. @dev comment removed